src/entitysdk/client.py

```diff
@@ -0,0 +1,255 @@
+"""Identifiable SDK client."""
+
+import httpx
+
+from entitysdk.common import ProjectContext
+from entitysdk.core import Identifiable
+from entitysdk.serdes import deserialize_entity, serialize_dict, serialize_entity
+from entitysdk.util import make_db_api_request
+
+
+class Client:
+    """Client for entitysdk."""
+
+    def __init__(
+        self,
+        api_url: str,
+        project_context: ProjectContext | None = None,
+        http_client: httpx.Client | None = None,
+    ) -> None:
+        """Initialize client.
+
+        Args:
+            api_url: The API URL to entitycore service.
+            project_context: Project context.
+            http_client: Optional HTTP client to use.
+        """
+        self.api_url = api_url
+        self.project_context = project_context
+        self._http_client = http_client
```

Comment:
After the last change the http client is not instantiated if not passed, but it can be
```suggestion
        self._http_client = http_client or httpx.Client()
```